middleware/2023-02-15-1.3.43.md

```diff
@@ -0,0 +1,40 @@
+---
+authors: poscreator
+slug: middleware/1.3.43
+tags: [Middleware, Germany]
+---
+
+# Middleware 1.3.43 (Germany)
+In this version of the Middleware we solved a bug concerning a sorting error in our DSFinV-K export. Additionally, we fixed a bug by changing back slash to slash in the path for FCC used on Linux.
+
+<!--truncate-->
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
+
+## Bug Fix: Reference Timespan-bugfix for DSFinV-K Export
+In case of items having the same timestamp there was a sorting error and items from lists were misplaced, so we corrected this.
+### Affected packages:
+- _fiskaltrust.Middleware.Queue.EF v1.3.43_
+- _fiskaltrust.Middleware.Queue.MySQL v1.3.43_
+- _fiskaltrust.Middleware.Queue.SQLite v1.3.43_
+
+## Bug Fix: Using slash instead of back slash in the path for FCC used on Linux
+Thanks to our partners we noticed that the path for FCC operating on Linux were using back slash instead of slash and we rectified that.
+
+### Affected packages:
+- _fiskaltrust.Middleware.SCU.DE.SwissbitCloud v1.3.43_
+- _fiskaltrust.Middleware.SCU.DE.DeutscheFiskal v1.3.43_
+
+## Next steps in the Middleware
+We will focus on developing of the next version of the fiskaltrust.Launcher in the next sprints.
```

Comment:
Should we change this to the IT middleware?

Reply:
good idea

Reply:
done